pkgs/development/tools/misc/binutils/libbfd.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv
+, gnu-config, autoreconfHook, bison, binutils-unwrapped, texinfo
+, libiberty, libintl, zlib
+}:
+
+stdenv.mkDerivation {
+  pname = "libbfd";
+  inherit (binutils-unwrapped) version src;
+
+  outputs = [ "out" "dev" ];
+
+  patches = binutils-unwrapped.patches ++ [
+    ./build-components-separately.patch
+  ];
+
+  # We just want to build libbfd
+  postPatch = ''
+    cd bfd
+  '';
```

Comment:
Should we use sourceRoot instead?

Reply:
`sourceRoot` would require us knowing the extracted path (which probably isn't a good idea to rely on), and will break patches.

Reply:
I think Sandro means cd $sourceRoot